src/site_apportionment/output_status_filtered.py

```diff
@@ -0,0 +1,56 @@
+"""The main file for the Outputs module."""
+import logging
+import pandas as pd
+from datetime import datetime
+from typing import Callable, Dict, Any
+
+
+StatusFilteredLogger = logging.getLogger(__name__)
+
+imp_markers_to_keep: list = ["R", "TMI", "CF", "MoR", "constructed"]
+marker_col: str = "imp_marker"
+
+def save_removed_markers(df: pd.DataFrame) -> pd.DataFrame:
+    """Filter rows neither clear nor imputed for output QA, based on imp_marker."""
+    to_remove = ~df[marker_col].isin(imp_markers_to_keep)
+    return df.copy().loc[to_remove]
+
+
+def output_status_filtered(
+    df: pd.DataFrame,
+    config: Dict[str, Any],
+    write_csv: Callable,
+    run_id: int,
+):
+    """Output rows that are neither clear nor imputed, before these are removed.
+
+    Args:
+        df (pd.DataFrame): Dataframe containing records filtered out from outputs.
+        config (dict): The configuration settings.
+        write_csv (Callable): Function to write to a csv file.
+         This will be the hdfs or network version depending on settings.
+        run_id (int): The current run id
+    """
+    StatusFilteredLogger.info("Starting status filtered output...")
+
+    # filter the dataframe
+    filtered_df = save_removed_markers(df)
+
+    NETWORK_OR_HDFS = config["global"]["network_or_hdfs"]
+    paths = config[f"{NETWORK_OR_HDFS}_paths"]
+    output_path = paths["output_path"]
+
+    tdate = datetime.now().strftime("%Y-%m-%d")
+    filename = f"status_filtered_qa_{tdate}_v{run_id}.csv"
+    write_csv(f"{output_path}/output_status_filtered_qa/{filename}", filtered_df)
+
+    StatusFilteredLogger.info("Finished status filtered output.")
+
+
+def keep_good_markers(
```

Comment:
add input arguments to the function definition: marker_col="imp_marker" and imp_markers_to_keep=["R", "TMI", "CF", "MoR", "constructed"].

I thought it was OK to pass them implicitly when the function was in the site_apportionment.py, but since it's in a different module, it needs arguments.

Reply:
I put them as globals in the other module, but I'll do what you suggest as otherwise we'd have to update in 2 places, not good.